src/lib/queries/getAccounts.ts

```diff
@@ -5,24 +5,23 @@ import type { AccountsMap } from '@/types/book';
  * Retrieves all the accounts and returns them in a map where the key
  * is the account guid.
  *
- * For first level accounts, the key is also the type of the account. For
- * example the root account can be found in `accounts.root`, the assets one
- * in `accounts.asset`.
+ * For first level accounts, the key is also the type of the account prefixed
+ * with type, same for the root account.
  */
 export default async function getAccounts(): Promise<AccountsMap> {
   const accounts = await Account.find();
 
   const accountsMap: AccountsMap = {};
   accounts.forEach(account => {
     if (account.type === 'ROOT' && !account.name.startsWith('Template')) {
-      accountsMap.root = account;
+      accountsMap.type_root = account;
```

Comment:
The actual fix. Note this has impact to how the map is used, now to get the root account you need to access `type_root`